prow/cmd/sinker/main.go

```diff
@@ -84,7 +84,8 @@ func clean(kc, pkc kubeClient, configAgent configAgent) {
 	}
 
 	// Now clean up old pods.
-	pods, err := pkc.ListPods(nil)
+	labels := map[string]string{kube.CreatedByProw: "true"}
+	pods, err := pkc.ListPods(labels)
```

Comment:
@spxtr if you have lots of pods laying around, you will have to manually delete them, or manually label them
```
kubectl label pods -n POD_NS --all created-by-prow=true
```

Reply:
Actually, this is a good candidate for including in the announcements section. Will update.